### ISSUE TITLE
MCA Reborn exclusions still show nameplate.

### ISSUE THREAD

**Issue by Wilevar** [Globox1997/Nameplate#2]
After using WTHIT to see that the Minecraft Comes Alive villagers and zombie villagers are:
 "mca:male_villager",
 "mca:female_villager",
 "mca:male__zombie_villager",
 "mca:female_zombie_villager",
 
 ... adding each of the above to the **nameplate.json5 "excluded_entities"** list didn't change anything. They still show a nameplate of "lvl # <CharacterName>" over each of their heads, which is overlapped by their MCA name plate when close enough to show it.

Fabric-api-0.56.2+1.19
Nameplate 1.0.3
RPGDifficulty-1.2.0
Minecraft-Comes-Alive-7.3.3+1.19-fabric

Copy of the **nameplate.json5** contents _(direct upload of .json5 files not supported, copy to txt uploaded)_:
[namplate_json.txt](https://github.com/Globox1997/Nameplate/files/8972357/namplate_json.txt)

Let me know if you need any additional information or testing done to narrow down the cause.

**Comment by Wilevar** [comment]
To deal with the overlap/conflict, I've temporarily disabled their name displays in MCA's JSON, but I figured you should know that the exclusions don't seem to be working.

**Comment by Globox1997** [comment]
Does a freshly spawned mob which is excluded have a nameplate?

**Comment by Globox1997** [comment]
Btw if you are using wthit I would disable the nameplates of mobs

**Comment by Wilevar** [comment]
> Does a freshly spawned mob which is excluded have a nameplate?

I didn't test it with a newly created villager, so it may function alright for newly spawned mobs. That said, it doesn't make much sense that a nameplate would be added to an existing mob if it's excluded, but not newly spawned ones.

I've since realized that the gradual increase in mob strength over time (via RPGDifficulty) would end up causing major problems for the MCA villagers. Since they're persistent, _their_ levels don't increase over time, so enemies would quickly outpace the guards ability to fight. Due to that, I removed RPGDifficulty and Namplate.

Regarding WTHIT, I had no issues/conflict there. MCA adds names above villagers heads, same as Nameplate does, so those two added elements were overlapping. WTHIT was never an issue. And given the way RPGDifficulty and Nameplate _seem_ to be intended to work, I assume they were never meant for use with persistent mobs with name tags from other sources.

Either way, it's no longer an issue for me directly and not one that anyone else has seen fit to mention, so I'll not bother you further with it. ^_^
